builtins/amp-pixel.js

```diff
@@ -72,6 +72,7 @@ export class AmpPixel extends BaseElement {
   /**
    * Triggers the signal.
    * @private
+   * @return {*} TODO: Specify return type
```

Comment:
This can return `{undefined}`, but we'll need to change `return this.triggerPromise_` to just `return`.

Reply:
I'm going to punt on this since it's a code change. See https://github.com/ampproject/amphtml/pull/23559#pullrequestreview-267861042